concourse/tasks/validate-integtest-result.yaml

```diff
@@ -0,0 +1,19 @@
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: gcr.io/gcp-guest/validate-integtest
```

Comment:
is there a reason to make this a container image, rather than using a generic image and running the script from a git resource?

Reply:
We need python3 and a package `junit2html`, so i build a new image with dependency installed.